src/Facebook/InstantArticles/Transformer/Transformer.php

```diff
@@ -42,6 +42,11 @@ class Transformer
     private static $allClassTypes = [];
 
     /**
+     * @var InstantArticle the initial context.
+     */
+    private $initialContext;
```

Comment:
Call it `$instantArticle`. Not every transformation needs to start with an InstantArticles, so it is not correct to assume this is the initial context.

What happens if one starts the transformation on a Paragraph instead? Nothing on this API actually enforces you to start you transformation on a InstantArticle.

In this case, this variable will be null, which is fine if we call it `$instantArticle` instead.